Cars/CarsRepository/Repo/CarRepository.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CarsRepository.Interface;
+using CarsData.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CarsRepository.Repo
+{
+    public class CarRepository : ICarRepository
+    {
+        private readonly AppDbContext appDbContext;
+
+        //EFDbContext context = new EFDbContext();
+        public CarRepository(AppDbContext appDbContext)
+        {
+            this.appDbContext = appDbContext;
```

Comment:
it's not really good to use this word ecerywhere, nane your class field like _appDbContext  and that's it